heareval/tasks/dcase2016_task2.py

```diff
@@ -88,7 +87,9 @@ def get_split_metadata(self, split: str) -> pd.DataFrame:
                 .replace("annotation", "sound")
                 .replace(".txt", ".wav")
             )
-            assert os.path.exists(sound_file)
+            # Remove the assert statement as this file might not exist.
```

Comment:
Is there a way to know if small mode is enabled? In which case we can run this when small is disabled.

Reply:
Yes. There is a way. But in general, we don't check this for all the tasks. I can add a general check in the pipeline for this. The check will run when small is false.